### DISCUSSION TITLE
[Question]: Non-openai format access problems

### DISCUSSION THREAD

**Discussion by 13531446481** [danny-avila/LibreChat#3078] [Q&A] [answered]
### What is your question?

I found that claude and other non-OpenAI models used their own company's reference format. What should I do if I am accessing a unified transfer to openai format, but I do not want to add a non-OpenAI model to the openai model list? Thank you for your answer

### More Details

I found that claude and other non-OpenAI models used their own company's reference format. What should I do if I am accessing a unified transfer to openai format, but I do not want to add a non-OpenAI model to the openai model list? Thank you for your answer

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Check out the documentation on Custom endpoints, this is the recommended way: 

https://www.librechat.ai/docs/quick_start/custom_endpoints

**Reply by 13531446481** [reply]
I wanted to implement the request of claude's model to the transfer station in openai format. I modified the configuration of 'endpoint' according to the document instructions. I saw that it said that if I wanted to use existing project icons, I need to define the endpoint name as one of the main endpoints. In this case, I changed the name to anthropic. Oddly enough, when I deploy, the command line prompts me to say that I cannot be one of the default endpoint, which is supposed to be different from what the documentation says. Moreover, when I tried to talk to the anthropic model in the application, I found that he seemed to send a request not to the transfer station I had given, but to the official website of anthropic. I would like to ask where there might be a problem with my configuration, or do I have a wrong understanding of some parts of the document? Thank you for your answer![image](https://github.com/danny-avila/LibreChat/assets/125166290/34cceeaa-1ec2-4cd8-b994-3c480f50f3f2)
![image](https://github.com/danny-avila/LibreChat/assets/125166290/4e9756d3-6c67-472f-a0ae-21264c18a69c)
![image](https://github.com/danny-avila/LibreChat/assets/125166290/dc5f9629-47b3-4581-81b0-9d80a9407a74)

**Reply by fuegovic** [reply]
Try with `iconurl:anthropic` instead and use a different `name` for the endpoint

**Reply by 13531446481** [reply]
Thank you, I found that after changing the name to Claude and then adding Claude in.env, I can select Claude's model when selecting model supplier, but I found that there seems to be a problem with the picture of the model. According to the document, I changed the iconURL to anthropic or the url of a picture, but the icon of the model will not be displayed correctly. I would like to ask what the problem is, thank you for your answer
![1719040176552](https://github.com/danny-avila/LibreChat/assets/125166290/c0d67dd1-5b3a-461f-8ac4-09551da22278)
![image](https://github.com/danny-avila/LibreChat/assets/125166290/83881cc8-4d9d-4aa0-ac94-1329e53bc587)

**Reply by 13531446481** [reply, marked as answer]
The image issue was fixed in the 2024.6.26 update

**Reply by R-Prady** [reply]
is there a way to add icons to the assets folder and use the folder path in the iconURL? im trying to inlcude nvidia endpoints and i need nvidia icon for it
